src/Command/CronDisable.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Command;
+
+use Magento\MagentoCloud\App\GenericException;
+use Magento\MagentoCloud\Cron\Switcher;
+use Magento\MagentoCloud\Util\BackgroundProcess;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * CLI command for disabling Magento cron processes
+ */
+class CronDisable extends Command
+{
+    public const NAME = 'cron:disable';
+
+    /**
+     * @var Switcher
+     */
+    private $cronSwitcher;
+
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @var BackgroundProcess
+     */
+    private $backgroundProcess;
+
+    /**
+     * @param Switcher $cronSwitcher
+     * @param BackgroundProcess $backgroundProcess
+     * @param LoggerInterface $logger
+     */
+    public function __construct(
+        Switcher $cronSwitcher,
+        BackgroundProcess $backgroundProcess,
+        LoggerInterface $logger
+    ) {
+        $this->cronSwitcher = $cronSwitcher;
+        $this->backgroundProcess = $backgroundProcess;
+        $this->logger = $logger;
+
+        parent::__construct();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function configure()
+    {
+        $this->setName(static::NAME)
+            ->setDescription('Disable all Magento cron processes');
+
+        parent::configure();
+    }
+
+    /**
+     * Runs process which finds all running Magento cron processes and kills them
```

Comment:
it also disable future cron processes for running

Reply:
the description has been updated